### PR TITLE
[DONE] Update create_pod_index.py

### PR DESCRIPTION
**Pull request by ptitloup**
Add try except when deleting index if not exist

# Before sending your pull request, make sure the following are done

* [x] You have read our [contribution guidelines](https://github.com/EsupPortail/Esup-Pod/blob/master/CONTRIBUTING.md).
* [x] Your PR targets the `develop` branch.
* [x] The title of your PR starts with `[WIP]` or `[DONE]`.

### CODE REVIEW

File: pod/video_search/management/commands/create_pod_index.py [hunk 1]
```diff
@@ -19,6 +16,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         """Create the Elasticsearch Pod index."""
-        delete_index_es()
+        try:
+            delete_index_es()
+            self.stdout.write(self.style.WARNING("The Pod index has been deleted.."))
```
Comment (Badatos) [inline pod/video_search/management/commands/create_pod_index.py:21]: Un seul point suffit ^^.

File: pod/video_search/management/commands/create_pod_index.py [hunk 2]
```diff
@@ -19,6 +16,10 @@ class Command(BaseCommand):
...
+        except exceptions.NotFoundError:
+            self.stdout.write(self.style.WARNING("Pod index not found."))
```
Comment (Badatos) [inline pod/video_search/management/commands/create_pod_index.py:23]: Je rajouterai "Pod index not found **on ElasticSearch server**."

**Review by Badatos — APPROVED** [review]
2 mini remarques, sinon OK pour moi ;)

File: pod/video_search/management/commands/create_pod_index.py (hunk 2 above)
Comment (ptitloup) [reply pod/video_search/management/commands/create_pod_index.py]: fait

File: pod/video_search/management/commands/create_pod_index.py (hunk 1 above)
Comment (ptitloup) [reply pod/video_search/management/commands/create_pod_index.py]: fait

**Pull request merged**